Tools/AP_Periph/Parameters.cpp

```diff
@@ -173,7 +173,7 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @RebootRequired: True
     GSCALAR(gps_port, "GPS_PORT", HAL_PERIPH_GPS_PORT_DEFAULT),
 
-#if HAL_NUM_CAN_IFACES >= 2
+#if HAL_NUM_CAN_IFACES >= 2 || GPS_MOVING_BASELINE
```

Comment:
Just checking for GPS_MOVING_BASELINE is good enough

Reply:
done